EZP-31912: [3.1] Fixed static server example and related pieces of File Management doc (#1148)
ezsystems/developer-documentation: docs/guide/file_management.md

                url_prefix: 'http://static.example.com/$var_dir$/$storage_dir$'
```

!!! caution

>     For security reasons it is recommended not to map `/path/to/ezplatform/public/` as a static server

Reviewer: ```suggestion
    For security reasons, do not map `/path/to/ezplatform/public/` as
```

>     Document Root. `/var/` directory should be mapped to `/path/to/ezplatform/public/var` directly instead.

Reviewer: ```suggestion
    Document Root of the static server.
    Map the `/var/` directory directly to `/path/to/ezplatform/public/var` instead.
```
